test/capypdftests.py

```diff
@@ -11,7 +11,14 @@
 except ModuleNotFoundError:
     sys.exit('PIL not found, test suite can not be run.')
 
-if shutil.which('gs') is None:
+source_root = pathlib.Path(__file__).parent.parent
```

Comment:
This seems to be a duplicated definition from below where all the path dirs are specified.